### PR TITLE
bump pytools and make corresponding changes

### PR DESCRIPTION
**Pull request by akspha**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: dags/suggestions.py [hunk 1]
```diff
@@ -68,7 +68,10 @@
 
 extract_conversation = BashOperator(
     task_id='extract_conversation',
-    bash_command='python3 -m tools.analysis.simple_stats --daily --upload_to_s3',
+    bash_command='python3 -m tools.analysis.simple_stats --daily --upload_to_s3 \
+    --start_date="{{ execution_date.format("%Y-%m-%d") }} 00:00:00" \
+    --end_date="{{ execution_date.format("%Y-%m-%d") }} 23:59:59" \
+    --timezone="America/Los_Angeles"',
```
Comment (JaekwanLee) [inline dags/suggestions.py:74]: Why do we need this? if needed, shouldn't it get from variable that we already have a timezone?

Comment (akspha) [reply dags/suggestions.py:74]: we made it mandatory, there is no variable for timezone we have it in airflow.cfg , can it be accessed, i searched the internet, didn't see a way

Comment (JaekwanLee) [reply dags/suggestions.py:74]: we have airflow variable for TIMEZONE

Comment (JaekwanLee) [reply dags/suggestions.py:74]: Ah.. this uses always PST/PDT. timezone shouldn't be a mandatory field. default UTC right?

Comment (akspha) [reply dags/suggestions.py:74]: so, python tools has UTC as default but Airflow is running in PST, so this should also be PST

Comment (akspha) [reply dags/suggestions.py:74]: so yes simple stats will use UTC if not supplied but we should supply PST because execution date is in PST from airflow

**Pull request merged**